darwin/importer/formats/csv_tags_video.py

```diff
@@ -51,15 +51,22 @@ def parse_path(path: Path) -> Optional[List[dt.AnnotationFile]]:
             file_annotation_map[filename].append(annotation)
     for filename in file_annotation_map:
         annotations = file_annotation_map[filename]
-        annotation_classes = {annotation.annotation_class for annotation in annotations}
+        annotation_classes = set(
+            [annotation.annotation_class for annotation in annotations]
+        )
```

Comment:
This could be simplified a bit by directly using set-comprehension:
```suggestion
        annotation_classes = set(
            annotation.annotation_class for annotation in annotations
        )
```